StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionNoResultsView.swift

```diff
@@ -0,0 +1,132 @@
+        verticalStackView.directionalLayoutMargins = NSDirectionalEdgeInsets(
+            top: 24,
+            leading: 24,
+            bottom: 0,
+            trailing: 24
+        )
+
+        let titleLabel = AttributedLabel(
+            font: .heading(.large),
+            textColor: .textDefault
+        )
+        titleLabel.textAlignment = .center
+        titleLabel.setText(
+            STPLocalizedString(
+                "No results",
+                "The title of a notice that appears at the bottom of search results. It appears when a user is searching for their bank, but no results are returned."
+            )
+        )
+        verticalStackView.addArrangedSubview(titleLabel)
+
+        let subtitleLabel = AttributedTextView(
+            font: .body(.medium),
+            boldFont: .body(.mediumEmphasized),
+            linkFont: .body(.mediumEmphasized),
+            textColor: .textDefault,
+            linkColor: .textActionPrimaryFocused,
+            showLinkUnderline: false,
+            alignCenter: true
+        )
+        if let didSelectManuallyEnterDetails = didSelectManuallyEnterDetails {
+            let subtitleText = STPLocalizedString(
+                "Try searching another bank or %@",
+                "Part of a subtitle of a notice that appears at the bottom of search results. It appears when a user is searching for their bank, but no results are returned. The '%@' will be replaced by a tappable text that says: 'manually enter details'."
+            )
+            let manuallyEnterDetailsText = STPLocalizedString(
+                "manually enter details",
+                "Part of a subtitle of a notice that appears at the bottom of search results. It appears when a user is searching for their bank, but no results are returned. This part of the notice will be tappable. Tapping this will allow a user to manually enter their bank details (account and routing numbers)."
+            )
+            subtitleLabel.setText(
+                String(
+                    format: subtitleText,
+                    "[\(manuallyEnterDetailsText)](stripe://this-url-will-be-ignored)"
+                ),
+                action: { _ in
+                    didSelectManuallyEnterDetails()
+                }
+            )
+        } else {
+            subtitleLabel.setText(
+                STPLocalizedString(
+                    "Try searching another bank",
+                    "The subtitle of a notice that appears at the bottom of search results. It appears when a user is searching for their bank, but no results are returned."
+                )
+            )
+        }
+        verticalStackView.addArrangedSubview(subtitleLabel)
+
+        addAndPinSubview(verticalStackView)
+
+        accessibilityIdentifier = "institution_no_results_footer_view"
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+#if DEBUG
```

Comment:
It's really cool to have a preview using swiftUI like this, I wonder if it's possible to create a shared lib? seems most of the previews needs sort of `makeUIView` and `updateUIView`

Reply:
this is just boiler plate for SwiftUI

one positive news is that I believe newer versions have fixed some of this via swift macros  so there's definitely chance for improvement in the future https://www.avanderlee.com/xcode/preview-swiftui-uikit-appkit-views/